.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
you can't use comments in json... you can get around it by the following convention:

"_comment": "comment text goes here..."

Reply:
Huh. This was autogenerated by vscode. I guess they strip this themselves.